### ISSUE TITLE
Errors in npm check (outdated)

### ISSUE THREAD

**Issue by arrmo** [homebridge/homebridge#3428]
### Analysis

Going to `/usr/lib/node_modules`, to check for outdated modules, I get an odd response 😆 ...
```
Package                 Current  Wanted  Latest  Location                     Depended by
homebridge-myq            3.3.0  2.12.0   3.3.0  node_modules/homebridge-myq  lib
homebridge-smartthings  MISSING   2.3.8   2.3.8  -                            lib

```

2 questions,
1. Why does myq show a wanted that is so old (and incorrect)? Not sure where this comes from.
2. smartthings is not even installed, so why is it being checked?

Thanks!

### Expected Behavior

The UI shows things up to date - so not expecting any outdated modules to be flagged.

### Steps To Reproduce

As above, 
1. cd /usr/lib/node_modules
2. npm outdated | grep homebridge

### Logs

```markdown
Information as above.
```

### Configuration

```json
The portion that applies?

    "disabledPlugins": [
        "homebridge-honeywell-home",
        "homebridge-tcc",
        "homebridge-nexia-thermostat-ts",
        "homebridge-nexia",
        "homebridge-smartthings"
    ]
```

### Environment

- OS: Ubuntu 22.04.3 LTS
- Software: 1.6.1
- Node: v18.16.0
- npm: 9.6.6

### Process Supervisor

systemd

**Comment by NorthernMan54** [comment]
Not sure that this is an homebridge issue, but an issue / feature of the npm application

**Comment by arrmo** [comment]
Was wondering that too! But I only see this for Homebridge items? Still may be npm, no argument 😆

**Comment by bwp91** [comment]
Can you go back into this folder `cd /usr/lib/node_modules` and display the output from running `npm list` ?

**Comment by ebaauw** [comment]
`npm` reads the current version from `package.json`.  This happens when that file is missing from `homebridge-smartthings`, usually because it is an empty directory or a symlink pointing to a directory that no longer exists.

**Comment by arrmo** [comment]
> Can you go back into this folder `cd /usr/lib/node_modules` and display the output from running `npm list` ?

Sure! Here you go,
```
lib@ /usr/lib
├── @babel/runtime-corejs3@7.21.0 extraneous
├── @cdmnky/nexia-api@0.0.2
├── @fastify/accept-negotiator@1.1.0 extraneous
├── @fastify/ajv-compiler@3.5.0 extraneous
├── @fastify/busboy@1.2.1 extraneous
├── @fastify/cors@8.0.0 extraneous
├── @fastify/deepmerge@1.3.0 extraneous
├── @fastify/error@3.2.0 extraneous
├── @fastify/fast-json-stringify-compiler@4.2.0 extraneous
├── @fastify/formbody@7.0.1 extraneous
├── @fastify/helmet@9.1.0 extraneous
├── @fastify/middie@8.0.0 extraneous
├── @fastify/multipart@7.1.0 extraneous
├── @fastify/static@6.5.0 extraneous
├── @fastify/swagger@7.4.1 extraneous
├── @homebridge/ciao@1.1.5 extraneous
├── @homebridge/dbus-native@0.5.0 extraneous
├── @homebridge/long@5.2.1 extraneous
├── @homebridge/put@0.0.8 extraneous
├── @leichtgewicht/ip-codec@2.0.4 extraneous
├── @nestjs/axios@0.1.0 extraneous
├── @nestjs/common@9.0.8 extraneous
├── @nestjs/core@9.0.8 extraneous
├── @nestjs/jwt@9.0.0 extraneous
├── @nestjs/mapped-types@1.1.0 extraneous
├── @nestjs/passport@9.0.0 extraneous
├── @nestjs/platform-fastify@9.0.8 extraneous
├── @nestjs/platform-socket.io@9.0.8 extraneous
├── @nestjs/swagger@6.0.5 extraneous
├── @nestjs/websockets@9.0.8 extraneous
├── @nuxtjs/opencollective@0.3.2 extraneous
├── @oznu/hap-client@1.9.0 extraneous
├── @types/component-emitter@1.2.11 extraneous
├── @types/cookie@0.4.1 extraneous
├── @types/cors@2.8.13 extraneous
├── @types/jsonwebtoken@8.5.8 extraneous
├── @types/node@18.15.12 extraneous
├── abstract-logging@2.0.1 extraneous
├── accepts@1.3.8 extraneous
├── ansi-regex@5.0.1 extraneous
├── aproba@1.2.0 extraneous
├── archy@1.0.0 extraneous
├── are-we-there-yet@1.1.7 extraneous
├── argparse@2.0.1 extraneous
├── array-flatten@2.1.2 extraneous
├── atomic-sleep@1.0.0 extraneous
├── available-typed-arrays@1.0.5 extraneous
├── avvio@8.2.1 extraneous
├── base64id@2.0.0 extraneous
├── big-integer@1.6.51 extraneous
├── binary@0.3.0 extraneous
├── bl@4.1.0 extraneous
├── bonjour-hap@3.6.4 extraneous
├── bonjour-service@1.1.1 extraneous
├── buffer-equal-constant-time@1.0.1 extraneous
├── buffer-indexof-polyfill@1.0.2 extraneous
├── buffer@5.7.1 extraneous
├── buffers@0.1.1 extraneous
├── call-bind@1.0.2 extraneous
├── chainsaw@0.1.0 extraneous
├── class-transformer@0.5.1 extraneous
├── class-validator@0.13.2 extraneous
├── cli-cursor@3.1.0 extraneous
├── cli-spinners@2.8.0 extraneous
├── clone@2.1.2 extraneous
├── code-point-at@1.1.0 extraneous
├── commander@5.1.0 extraneous
├── component-emitter@1.3.0 extraneous
├── concat-map@0.0.1 extraneous
├── consola@2.15.3 extraneous
├── console-control-strings@1.1.0 extraneous
├── content-disposition@0.5.4 extraneous
├── cookie@0.5.0 extraneous
├── core-js-pure@3.30.1 extraneous
├── cors@2.8.5 extraneous
├── cron-parser@3.5.0 extraneous
├── dayjs@1.11.4 extraneous
├── decamelize@3.2.0 extraneous
├── decompress-response@4.2.1 extraneous
├── deep-equal@2.2.0 extraneous
├── deep-extend@0.6.0 extraneous
├── deep-is@0.1.4 extraneous
├── deepmerge@4.3.1 extraneous
├── defaults@1.0.4 extraneous
├── define-properties@1.2.0 extraneous
├── delegates@1.0.0 extraneous
├── depd@2.0.0 extraneous
├── destroy@1.2.0 extraneous
├── detect-libc@1.0.3 extraneous
├── dns-equal@1.0.0 extraneous
├── dns-packet@5.6.0 extraneous
├── duplexer@0.1.2 extraneous
├── duplexer2@0.1.4 extraneous
├── ecdsa-sig-formatter@1.0.11 extraneous
├── ee-first@1.1.1 extraneous
├── encodeurl@1.0.2 extraneous
├── engine.io-parser@5.0.6 extraneous
├── engine.io@6.2.1 extraneous
├── es-get-iterator@1.1.3 extraneous
├── escape-html@1.0.3 extraneous
├── etag@1.8.1 extraneous
├── event-stream@4.0.1 extraneous
├── expand-template@2.0.3 extraneous
├── fast-decode-uri-component@1.0.1 extraneous
├── fast-json-stringify@5.7.0 extraneous
├── fast-querystring@1.1.1 extraneous
├── fast-redact@3.1.2 extraneous
├── fast-safe-stringify@2.1.1 extraneous
├── fast-srp-hap@2.0.4 extraneous
├── fast-uri@2.2.0 extraneous
├── fastify-plugin@3.0.1 extraneous
├── fastify@4.3.0 extraneous
├── fastq@1.15.0 extraneous
├── find-my-way@7.6.0 extraneous
├── follow-redirects@1.15.2 extraneous
├── for-each@0.3.3 extraneous
├── form-data@4.0.0 extraneous
├── forwarded@0.2.0 extraneous
├── fresh@0.5.2 extraneous
├── from@0.1.7 extraneous
├── fs-constants@1.0.0 extraneous
├── fs-extra@10.1.0 extraneous
├── fstream@1.0.12 extraneous
├── function-bind@1.1.1 extraneous
├── functions-have-names@1.2.3 extraneous
├── futoin-hkdf@1.4.3 extraneous
├── gauge@2.7.4 extraneous
├── get-intrinsic@1.2.0 extraneous
├── github-from-package@0.0.0 extraneous
├── gopd@1.0.1 extraneous
├── graceful-fs@4.2.11 extraneous
├── hap-nodejs@0.11.0 extraneous
├── has-bigints@1.0.2 extraneous
├── has-property-descriptors@1.0.0 extraneous
├── has-symbols@1.0.3 extraneous
├── has-tostringtag@1.0.0 extraneous
├── has-unicode@2.0.1 extraneous
├── has@1.0.3 extraneous
├── helmet@5.1.1 extraneous
├── hexoid@1.0.0 extraneous
├── hexy@0.2.11 extraneous
├── homebridge-blink-for-home@3.9.2 (git+ssh://git@github.com/arrmo/homebridge-blink-for-home.git#f889873affec8708cdac4ce699a6da539a528fb6)
├── homebridge-config-ui-x@4.50.5
├── homebridge-myq@3.3.0
├── homebridge-smartthings-ik@1.5.14 extraneous
├── homebridge-tplink-smarthome@7.0.1
├── homebridge@1.6.1
├── http-errors@2.0.0 extraneous
├── inflection@1.13.4 extraneous
├── ini@1.3.8 extraneous
├── internal-slot@1.0.5 extraneous
├── ip-regex@4.3.0 extraneous
├── ip@1.1.8 extraneous
├── ipaddr.js@1.9.1 extraneous
├── is-arguments@1.1.1 extraneous
├── is-array-buffer@3.0.2 extraneous
├── is-bigint@1.0.4 extraneous
├── is-boolean-object@1.1.2 extraneous
├── is-callable@1.2.7 extraneous
├── is-date-object@1.0.5 extraneous
├── is-fullwidth-code-point@1.0.0 extraneous
├── is-interactive@1.0.0 extraneous
├── is-map@2.0.2 extraneous
├── is-nan@1.3.2 extraneous
├── is-number-object@1.0.7 extraneous
├── is-regex@1.1.4 extraneous
├── is-set@2.0.2 extraneous
├── is-shared-array-buffer@1.0.2 extraneous
├── is-string@1.0.7 extraneous
├── is-symbol@1.0.4 extraneous
├── is-typed-array@1.1.10 extraneous
├── is-unicode-supported@0.1.0 extraneous
├── is-url@1.2.4 extraneous
├── is-weakmap@2.0.1 extraneous
├── is-weakset@2.0.2 extraneous
├── is2@2.0.9 extraneous
├── isarray@2.0.5 extraneous
├── iterare@1.2.1 extraneous
├── js-yaml@4.1.0 extraneous
├── json-schema-resolver@1.3.0 extraneous
├── jsonfile@6.1.0 extraneous
├── jsonwebtoken@8.5.1 extraneous
├── libphonenumber-js@1.10.28 extraneous
├── light-my-request@5.3.0 extraneous
├── listenercount@1.0.1 extraneous
├── lodash.includes@4.3.0 extraneous
├── lodash.isboolean@3.0.3 extraneous
├── lodash.isinteger@4.0.4 extraneous
├── lodash.isnumber@3.0.3 extraneous
├── lodash.isplainobject@4.0.6 extraneous
├── lodash.isstring@4.0.1 extraneous
├── lodash.once@4.1.1 extraneous
├── lodash@4.17.21 extraneous
├── log-symbols@4.1.0 extraneous
├── long-timeout@0.1.1 extraneous
├── luxon@1.28.1 extraneous
├── map-stream@0.0.7 extraneous
├── mime@1.6.0 extraneous
├── mimic-response@2.1.0 extraneous
├── mkdirp-classic@0.5.3 extraneous
├── mkdirp@0.5.6 extraneous
├── multicast-dns-service-types@1.1.0 extraneous
├── multicast-dns@7.2.5 extraneous
├── n@9.1.0
├── nan@2.17.0 extraneous
├── napi-build-utils@1.0.2 extraneous
├── negotiator@0.6.3 extraneous
├── node-abi@2.30.1 extraneous
├── node-cache@5.1.2 extraneous
├── node-fetch@2.6.9 extraneous
├── node-persist@0.0.11 extraneous
├── node-pty-prebuilt-multiarch@0.10.1-pre.5 extraneous
├── node-schedule@2.1.0 extraneous
├── npm@10.0.0
├── npmlog@4.1.2 extraneous
├── number-is-nan@1.0.1 extraneous
├── object-assign@4.1.1 extraneous
├── object-hash@3.0.0 extraneous
├── object-inspect@1.12.3 extraneous
├── object-is@1.1.5 extraneous
├── object-keys@1.1.1 extraneous
├── object.assign@4.1.4 extraneous
├── on-exit-leak-free@2.1.0 extraneous
├── on-finished@2.4.1 extraneous
├── openapi-types@11.1.0 extraneous
├── ora@5.3.0 extraneous
├── p-limit@3.1.0 extraneous
├── passport-jwt@4.0.0 extraneous
├── passport-strategy@1.0.0 extraneous
├── passport@0.6.0 extraneous
├── path-is-absolute@1.0.1 extraneous
├── path-to-regexp@3.2.0 extraneous
├── pause-stream@0.0.11 extraneous
├── pause@0.0.1 extraneous
├── pino-abstract-transport@1.0.0 extraneous
├── pino-std-serializers@6.2.0 extraneous
├── pino@8.11.0 extraneous
├── prebuild-install@6.1.4 extraneous
├── process-warning@2.2.0 extraneous
├── proxy-addr@2.0.7 extraneous
├── pump@3.0.0 extraneous
├── q@1.1.2 extraneous
├── qrcode-terminal@0.12.0 extraneous
├── quick-format-unescaped@4.0.4 extraneous
├── range-parser@1.2.1 extraneous
├── rc@1.2.8 extraneous
├── readable-stream@4.3.0 extraneous
├── real-require@0.2.0 extraneous
├── reflect-metadata@0.1.13 extraneous
├── regenerator-runtime@0.13.11 extraneous
├── regexp.prototype.flags@1.5.0 extraneous
├── restore-cursor@3.1.0 extraneous
├── ret@0.2.2 extraneous
├── reusify@1.0.4 extraneous
├── rimraf@2.7.1 extraneous
├── safe-regex2@2.0.0 extraneous
├── safe-stable-stringify@2.4.3 extraneous
├── sax@1.2.4 extraneous
├── secure-json-parse@2.7.0 extraneous
├── send@0.18.0 extraneous
├── set-blocking@2.0.0 extraneous
├── set-cookie-parser@2.6.0 extraneous
├── setimmediate@1.0.5 extraneous
├── setprototypeof@1.2.0 extraneous
├── side-channel@1.0.4 extraneous
├── simple-get@3.1.1 extraneous
├── socket.io-adapter@2.4.0 extraneous
├── socket.io-parser@4.0.5 extraneous
├── socket.io@4.5.1 extraneous
├── sonic-boom@3.3.0 extraneous
├── sorted-array-functions@1.3.0 extraneous
├── source-map-support@0.5.21 extraneous
├── source-map@0.6.1 extraneous
├── split@1.0.1 extraneous
├── statuses@2.0.1 extraneous
├── stop-iteration-iterator@1.0.0 extraneous
├── stream-combiner@0.2.2 extraneous
├── stream-wormhole@1.1.0 extraneous
├── string-width@1.0.2 extraneous
├── strip-ansi@6.0.1 extraneous
├── strip-json-comments@2.0.1 extraneous
├── swagger-ui-dist@4.13.2 extraneous
├── tail@2.2.4 extraneous
├── tar-fs@2.1.1 extraneous
├── tar-stream@2.2.0 extraneous
├── tcp-port-used@1.0.2 extraneous
├── text-decoding@1.0.0 extraneous
├── thread-stream@2.3.0 extraneous
├── through@2.3.8 extraneous
├── thunky@1.1.0 extraneous
├── tiny-lru@8.0.2 extraneous
├── toidentifier@1.0.1 extraneous
├── tr46@0.0.3 extraneous
├── traverse@0.3.9 extraneous
├── tweetnacl@1.0.3 extraneous
├── universalify@2.0.0 extraneous
├── unzipper@0.10.11 extraneous
├── utils-merge@1.0.1 extraneous
├── uuid@8.3.2 extraneous
├── validator@13.9.0 extraneous
├── vary@1.1.2 extraneous
├── wcwidth@1.0.1 extraneous
├── webidl-conversions@3.0.1 extraneous
├── whatwg-url@5.0.0 extraneous
├── which-boxed-primitive@1.0.2 extraneous
├── which-collection@1.0.1 extraneous
├── which-typed-array@1.1.9 extraneous
├── wide-align@1.1.5 extraneous
├── xml2js@0.4.23 extraneous
├── xmlbuilder@11.0.1 extraneous
├── xregexp@4.4.1 extraneous
└── yocto-queue@0.1.0 extraneous
```

> `npm` reads the current version from `package.json`. This happens when that file is missing from `homebridge-smartthings`, usually because it is an empty directory or a symlink pointing to a directory that no longer exists.

That may be it? I uninstalled `homebridge-smartthings`, no longer having the issue now?

Thanks!
